src/main/java/io/github/townyadvanced/townycombat/utils/TownyCombatItemUtil.java

```diff
@@ -138,9 +155,10 @@ else if (currentCraftingMatrix[i].getAmount() > 1
      *
      * @param itemHolder the holder of the item
      * @param offHand Is the item in offhand?, otherwise it is main hand.
-     * @param normalizedChance the chance
+     * @param percentageChange the chance
      */
-    public static void rollBreakItemInHand(Player itemHolder, boolean offHand, double normalizedChance) {
+    public static void rollBreakItemInHand(Player itemHolder, boolean offHand, double percentageChange) {
```

Comment:
normalizedChance got turned into percentageChan**g**e.

Reply:
That text change is intentional, 
because the method now takes in a percentage (e.g 15) rather than a normalized number (e.g 0.15)